modules/ppcp-button/resources/js/modules/Helper/ButtonDisabler.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @param selectorOrElement
+ * @returns {Element}
+ */
+const getElement = (selectorOrElement) => {
+    if (typeof selectorOrElement === 'string') {
+        return document.querySelector(selectorOrElement);
+    }
+    return selectorOrElement;
+}
+
+export const setEnabled = (selectorOrElement, enable, form = null) => {
+    const element = getElement(selectorOrElement);
+
+    if (!element) {
+        return;
+    }
+
+    if (enable) {
+        jQuery(element).css({
```

Comment:
I think PayPal SDK supports disabling/enabling? https://developer.paypal.com/docs/checkout/standard/customize/validate-user-input/
As I understand you can keep the `actions` reference from `onInit` somewhere and call its methods when needed.

Reply:
I will test the method you suggested..

Reply:
By disabling with PayPal actions they aren't greyed out. I'll implement an hybrid solution.